app/helpers/markdown_helper.rb

```diff
@@ -4,13 +4,41 @@ module MarkdownHelper
   def camofy(markdown)
     return unless markdown
 
-    markdown.gsub(markdown_img_regex) do
+    markdown = sub_markdown(markdown)
```

Comment:
why are you using the variable name markdown

Reply:
I could change it to `text` if that scratches an itch for you. You're correct in suggesting this, I was just lazy

Reply:
it is a small change and it improve readabilty